src/NDItk/multigroup/AverageFissionEnergyRelease.hpp

```diff
@@ -0,0 +1,97 @@
+#ifndef NJOY_NDITK_MULTIGROUP_AVERAGEFISSIONENERGYRELEASE
+#define NJOY_NDITK_MULTIGROUP_AVERAGEFISSIONENERGYRELEASE
+
+// system includes
+
+// other includes
+#include "tools/Log.hpp"
+#include "NDItk/base/RealListRecord.hpp"
+
+namespace njoy {
+namespace NDItk {
+namespace multigroup {
+
+/**
+ *  @brief An average fission energy release record for multigroup
+ *         neutron and photon data
+ */
+class AverageFissionEnergyRelease : protected base::RealListRecord {
+
+  /* fields */
+
+  /* auxiliary functions */
+
+public:
+
+  /* constructor */
+
+  #include "NDItk/multigroup/AverageFissionEnergyRelease/src/ctor.hpp"
+
+  /* methods */
```

Comment:
Can we also have one to return all six as a vector?

Reply:
Yes, that is the values() method that is taken over from the base class.